File: src/client/rightsEnvelope.js
const HOST_DID = "bostrom18sd2ujv24ual9c9pshtxys6j8knh6xaead9ye7";
const RIGHTS_PROFILE_ID = "host-rights-travel-us-maricopa.v1";

export function buildRightsEnvelope({ queryId, platformLabel }) {
  return {
    query_id: queryId,
    aichat_platform: platformLabel,
    did: HOST_DID,
    rights_profile_id: RIGHTS_PROFILE_ID,
    mode: "propose-only"
  };
}

File: java/augdoctor-rights/src/main/java/org/augdoctor/RightsEnvelope.java
package org.augdoctor;

public final class RightsEnvelope {
    public static final String HOST_ID =
            "bostrom18sd2ujv24ual9c9pshtxys6j8knh6xaead9ye7";
    public static final String RIGHTS_PROFILE_ID =
            "host-rights-travel-us-maricopa.v1";

    private RightsEnvelope() {}

    public static String hostId() {
        return HOST_ID;
    }

    public static String rightsProfileId() {
        return RIGHTS_PROFILE_ID;
    }
}
